RecoTauTag/Configuration/python/tools/dumpTauVariables_cfg.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("DUMP")
```

Comment:
Hi David, 
I had not realized that the move of files was incomplete. This should be the case now. I'll make a corresponding comment at the end of this conversation.